api/src/main/java/cz/muni/fi/pa165/mushrooms/dto/MushroomHunterCreateDTO.java

```diff
@@ -0,0 +1,75 @@
+package cz.muni.fi.pa165.mushrooms.dto;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * The DTO to be used at user creation.
+ *
+ * @author bkompis
+ */
+public class MushroomHunterCreateDTO {
+    private String userNickname;
+    private String unencryptedPassword;
+    private String firstName;
+    private String surname;
+    private String personalInfo;

```

Comment:
maybe we can use some constrains like @NotNull here

Reply:
will do